src/test/kotlin/com/vanniktech/rxriddles/solutions/Riddle37Solution.kt

```diff
@@ -0,0 +1,12 @@
+package com.vanniktech.rxriddles.solutions
+
+import io.reactivex.Observable
+import java.io.IOException
+
+object Riddle37Solution {
+  fun solve(source: Observable<Boolean>): Observable<Boolean> {
+    return source.onErrorResumeNext { t: Throwable ->
```

Comment:
just use `it`

Reply:
I don't think we can use `it` here because it can't infer whether we are invoking `ObservableSource<? extends T>` or `Function<? super Throwable, ? extends ObservableSource<? extends T>>`